Source/Core/VideoCommon/FramebufferManager.cpp

```diff
@@ -409,6 +410,8 @@ MathUtil::Rectangle<int> FramebufferManager::GetEFBCacheTileRect(u32 tile_index)
 
 u32 FramebufferManager::PeekEFBColor(u32 x, u32 y)
 {
+  LWDProfiler::ScopedMarker marker("FBuffer/Peek EFB Color");
```

Comment:
Move this (and depth) to PouplateEFBCache, which is the actual expensive operation.

Reply:
Flushing the readback texture is the slow part of EFB readbacks and that can happen in `PeekEFBColor`.